yamllint/rules/__init__.py

```diff
@@ -63,8 +65,11 @@
 }
 
 
-def get(id):
-    if id not in _RULES:
+def get(id, rules=None):
```

Comment:
If nobody uses the optional parameter `rules=None`, I'm for removing it.

Reply:
I'm not sure but it might be better to keep it as is or allow passing some arguments to make easier for unit tests, AFAIK.

Reply:
To make tests easier and leave a room to control the rules' mappings to search for, those are necessary, I think.Please take a look at the commit 01554f2 also.